### PR TITLE
doc: add document for HTTP Header

### PR DESCRIPTION
**Pull request by qingbozhang**
This PR reopen for #4901

### CODE REVIEW

**Comment by qingbozhang** [comment]
Hi @jpkrohling I reopen a PR here. Thanks for review. [link](https://github.com/open-telemetry/opentelemetry-collector/pull/4941)

**Review by jpkrohling — APPROVED** [review]

**Comment by qingbozhang** [comment]
Hi @codeboten, please add ***Skip Changelog*** label again. Thanks

**Review by hickeyma — APPROVED** [review]
LGTM, thanks @qingbozhang

File: config/confighttp/README.md [hunk 1]
```diff
@@ -71,6 +71,9 @@ will not be enabled.
 [cors-headers]: https://developer.mozilla.org/en-US/docs/Glossary/CORS-safelisted_request_header
 [cors-cache]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age
 [origin]: https://developer.mozilla.org/en-US/docs/Glossary/Origin
+[attribute-processor]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/attributesprocessor/README.md
+
+You can enable [`attribute processor`][attribute-processor] to append any http header to span's attribute using custom key. You also need to enable the "include_metadata"
```
Comment (bogdandrutu) [inline config/confighttp/README.md:76]: How is the attribute processor related to this section of `cors`?

Comment (jpkrohling) [reply config/confighttp/README.md:76]: You mean on line 74? It doesn't, you are right. But the text itself is on a good position, IMO.

Comment (bogdandrutu) [reply config/confighttp/README.md:76]: oh... these are links :))) 

Can we move them to the bottom of the file so easier to remember/identify that these are links used in the doc?

**Review by bogdandrutu — APPROVED** [review]

File: config/confighttp/README.md (hunk 1 above)
Comment (jpkrohling) [reply config/confighttp/README.md:76]: Good idea.

Comment (jpkrohling) [reply config/confighttp/README.md:76]: @qingbozhang, could you do this change?

Comment (qingbozhang) [reply config/confighttp/README.md:76]: OK, no problem.  just move line 74 or move line 70~74?  maybe 70~74 IMO.

**Comment by qingbozhang** [comment]
I moved links to bottom.[960fb5e](https://github.com/open-telemetry/opentelemetry-collector/pull/5070/commits/960fb5e5aceb96197dea172a86ef5857fadbbf31) Please review. Thanks

**Review by jpkrohling — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Pull request merged**